src/Microsoft.DotNet.Build.Tasks/PackageFiles/resources.targets

```diff
@@ -20,20 +20,25 @@
           GenerateResourcesSource;
           $(CompileDependsOn);
       </CompileDependsOn>
+      <ResourcesFolderPath Condition="'$(BuildingUAPVertical)' == 'true'">$(RuntimePath)resw</ResourcesFolderPath>
```

Comment:
We should then find a different place but I don't think this is the right one. Also, if we want to put it here, we should at least make this overridable by checking if it's set already.